src/components/application_manager/rpc_plugins/rc_rpc_plugin/src/rc_app_extension.cc

```diff
@@ -229,6 +226,22 @@ std::set<ModuleUid> RCAppExtension::InteriorVehicleDataSubscriptions() const {
   return subscribed_interior_vehicle_data_;
 }
 
+bool RCAppExtension::AddPendingSubscription(const ModuleUid& module) {
+  return pending_subscriptions_.insert(module).second;
+}
+
+void RCAppExtension::RemovePendingSubscription(const ModuleUid& module) {
+  subscribed_interior_vehicle_data_.erase(module);
```

Comment:
@SNiukalov why it called _RemovePendingSubscription_ but we removing from _subscribed_interior_vehicle_data__?
shouldn't it be pending_subscriptions_.erase(module); ?

Reply:
@LitvinenkoIra 
Fixed in:https://github.com/LuxoftSDL/sdl_core/pull/129/commits/76ecf5e3dcc3795b8559ff62193dbf46fc2bdba7